lib/Search/UI/SearchResultsController.php

```diff
@@ -87,6 +87,27 @@ public function __construct(SearchQuery $query, SearchResults $results)
     public function display()
     {
         $headers = $this->getListViewHeaders();
```

Comment:
Unhanded exception here

Reply:
I actually did not change this line.

Reply:
Btw, It's handled at https://github.com/salesagility/SuiteCRM/blob/develop/modules/Home/Search.php#L52